packages/documentation/src/pages/getting-started/common-tasks/lint.mdx

```diff
@@ -1,23 +0,0 @@
-# Linting your code
```

Comment:
Consolidated this into "Run tests" document. I think it's okay to consider linting a form of testing- it validates more than formatting like consistent variable declarations and other issues that may only manifest during runtime.